web/src/engine/websites/MangaCrab.ts

```diff
@@ -10,10 +10,18 @@ const chapterScript = `
     }));
 `;
 
+const pageScript = `
+    [...document.querySelectorAll('div.page-break img.wp-manga-chapter-img[id^="image-"]')]
```

Comment:
Suggestion to use URL pattern matching of attribute values instead of possible rotating attribute name

```js
[...document.querySelectorAll('div.page-break img.wp-manga-chapter-img[id^="image-"]')]
    .map(img => [...img.attributes].find(attribute => attribute.value.startsWith('/validate2.php'))?.value)
    .filter(img => img);
```